app/src/main/java/bankingapp/LogIn.java

```diff
@@ -91,29 +40,34 @@ public void checkAccount(Connection con, String accountNumber, String pinNumber)
                     "5. Log Out\n" +
                     "0. Exit");
 
-            answer = scanner.nextInt();
+            answer = input.nextInt();
 
             switch (answer) {
                 case 1:
-                    int balance = DBTrans.checkBalance(con, accountNumber);
+                    int balance = db.checkBalance(accountNumber);
                     System.out.println("\nBalance: " + balance);
                     break;
 
                 case 2:
                     System.out.println("Enter income:");
-                    income = scanner.nextInt();
-                    DBTrans.addIncome(con, income, accountNumber);
+                    income = input.nextInt();
+                    db.addIncome(income, accountNumber);
                     System.out.println("Income was added!");
                     break;
 
                 case 3:
-                    DBTrans.transferFunds(con, accountNumber);
+                    App.transferFunds(db,con, accountNumber);
                     break;
 
                 case 4:
-                    DBTrans.deleteAcc(con, accountNumber);
-                    System.out.println("The account has been closed!");
-                    break subMenu;
+                    if(db.checkBalance(accountNumber) > 0){
```

Comment:
Great catch!

Bugfixes deserve a separate commit. In fact, most teams will require a separate PR, even for a change as small as this. It allows you to make decisions about the bugfix separately from new features, and allows you and your team to assign them different priorities and release schedules.